Droid/Playing/PollingService.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Xamarin.Forms;
+
+namespace MeNext.Droid
+{
+	/// <summary>
+	/// Class which handles background polling for Android
+	/// </summary>
+	[Service]
+	public class PollingService : Service
```

Comment:
Forms tasks only work while the app is open, so the host wouldn't be able to continue playing music if the app were closed. Only the native tasks have this flexibility (even though I haven't actually configured it yet).